WooCommerce/Classes/ViewRelated/Dashboard/Settings/Plugins/PluginListViewController.swift

```diff
@@ -3,15 +3,18 @@ import WordPressUI
 
 /// View Controller for the Plugin List Screen.
 ///
-final class PluginListViewController: UIViewController {
+final class PluginListViewController: UIViewController, GhostableViewController {
 
     private let viewModel: PluginListViewModel
 
     @IBOutlet private var tableView: UITableView!
 
-    /// Separate table view for ghost animation.
-    ///
-    private let ghostTableView = UITableView(frame: .zero, style: .grouped)
+    lazy var ghostTableViewController = GhostTableViewController(options: GhostTableViewOptions(cellClass: HeadlineLabelTableViewCell.self,
+                                                                                                rowsPerSection: [10],
+                                                                                                tableViewStyle: .grouped,
+                                                                                                backgroundColor: .listBackground,
+                                                                                                separatorStyle: .singleLine,
```

Comment:
It seems to me that `.listBackground` and `.singleLine` are the most common values for background color and separtor style - would it be more convenient to set them as the default values so that we can omit these in the setup?

Reply:
@itsmeichigo Thanks a lot for the review and the hint! Indeed, they are the most common values, I changed it in https://github.com/woocommerce/woocommerce-ios/pull/6535/commits/305eac306a7691456dfe357b30e6de3b9b2998a3